app/models/device.rb

```diff
@@ -0,0 +1,79 @@
+class Device < ApplicationRecord
+  belongs_to :user
+
+  IOS = "iOS".freeze
+  ANDROID = "Android".freeze
+
+  validates :token, uniqueness: { scope: %i[user_id platform app_bundle] }
+
+  def create_notification(title, body, payload)
+    case platform
+    when IOS
+      ios_notification(title, body, payload)
+    when ANDROID
+      android_notification(title, body, payload)
+    end
+  end
+
+  private
+
+  def ios_notification(title, body, payload)
+    n = Rpush::Apns2::Notification.new
+    # [@forem/backend] `.where().first` is necessary because we use Redis data storage
+    # https://github.com/rpush/rpush/wiki/Using-Redis#find_by_name-cannot-be-used-in-rpush-redis
+    # rubocop:disable Rails/FindBy
+    n.app = Rpush::Apns2::App.where(name: app_bundle).first || recreate_ios_app!
+    # rubocop:enable Rails/FindBy
+
+    n.device_token = token
+    n.data = {
+      aps: {
+        :alert => {
+          title: ApplicationConfig["COMMUNITY_NAME"],
+          subtitle: title,
+          body: body
+        },
+        "thread-id" => ApplicationConfig["COMMUNITY_NAME"]
+      },
+      data: payload
+    }
```

Comment:
This hash has symbol keys, a symbol key with a hash rocket, and a string key. Does it make sense to make them all the same style or is there additional meaning here that I'm overlooking? 🙂

Reply:
"New York's hottest nightclub has it all: symbol keys, a symbol key with a hash rocket, and a string key" 😆 

Simplified this to all symbol keys ✅